From 926d3e14876929d027a46647298cc113aa034e1c Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Mon, 21 Jun 2021 21:29:48 -0700
Subject: [PATCH 133/196] Bump version to 1.0.1 (#210)

* Fixes a problem with the seeder segfaulting when there are no
  checkpoints.
* Fixes the input dialog validation to allow lotus mainnet addresses.
---
 CMakeLists.txt                | 2 +-
 contrib/aur/lotus-qt/PKGBUILD | 2 +-
 contrib/aur/lotus/PKGBUILD    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c673dba5..cebda8f9e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 1.0.0
+	VERSION 1.0.1
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://www.givelotus.org"
 )
diff --git a/contrib/aur/lotus-qt/PKGBUILD b/contrib/aur/lotus-qt/PKGBUILD
index ab7bb30ea..805fecec7 100644
--- a/contrib/aur/lotus-qt/PKGBUILD
+++ b/contrib/aur/lotus-qt/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotus-qt
-pkgver=1.0.0
+pkgver=1.0.1
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-cli, lotus-tx, lotus-seeder and lotus-qt"
 arch=('i686' 'x86_64')
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index 10accd250..ddedc97f3 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=1.0.0
+pkgver=1.0.1
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

